config/environments/test.rb

```diff
@@ -5,8 +5,10 @@
   config.public_file_server.headers = { 'Cache-Control' => 'public, max-age=3600' }
   config.consider_all_requests_local       = true
   config.action_controller.perform_caching = false
+  config.cache_store = :null_store
```

Comment:
Why not turn this off, right?

Reply:
[It defaults to `:file_store`](https://guides.rubyonrails.org/configuring.html). I don't know exactly what that will do, but I think we don't want any caching in tests so I would lean to say let's keep it `:null_store`